lib/syscontrol/syscontrol.go

```diff
@@ -71,20 +72,21 @@ curl -i -XPOST 'http://127.0.0.1:8086/debug/ctrl?mod=force_broadcast_query&enabl
 */
```

Comment:
Add the usage for `mod=time_filter_protection`.

Reply:
> Add the usage for `mod=time_filter_protection`.

ok